### PR TITLE
Serverless updates for OCP 4.5+ docs

### PR DESCRIPTION
**Pull request by abrennan89**
- Removed pre-OCP 4.5 release notes (pre-1.7.0)
- Updated upgrade docs to remove anything pre-1.7.0; updated channel to `4.5`
- Improvements to Support section

### CODE REVIEW

**Comment by alanfx** [comment]
These changes all look good to me, @abrennan89! 
QE Approve

**Comment by abrennan89** [comment]
Peer review requested

File: serverless/serverless-release-notes.adoc [hunk 1]
```diff
@@ -8,17 +8,14 @@ toc::[]
...
 
 // Modules included, most to least recent
+// include::modules/serverless-rn-1-8-0.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-7-2.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-7-1.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-7-0.adoc[leveloffset=+1]
-include::modules/serverless-rn-1-6-0.adoc[leveloffset=+1]
-include::modules/serverless-rn-1-5-0.adoc[leveloffset=+1]
 
 [id="additional-resources"]
```
Comment (adellape) [inline serverless/serverless-release-notes.adoc:20]: Not related this PR's changes really, but would recommend making this ID more unique, e.g., `serverless-release-notes-additional-resources`.

**Review by adellape — COMMENTED** [review]
:+1:

File: serverless/serverless-release-notes.adoc (hunk 1 above)
Comment (abrennan89) [reply serverless/serverless-release-notes.adoc:20]: Cool, I'll do that in another PR that applies for earlier versions too though if that's OK :slightly_smiling_face: adding it to my TODO list.

**Pull request merged**